src/Components/Ignitor/src/ElementNode.cs

```diff
@@ -88,7 +88,7 @@ internal Task SelectAsync(HubConnection connection, string value)
             return DispatchEventCore(connection, Serialize(webEventDescriptor), Serialize(args));
         }
 
-        internal Task ClickAsync(HubConnection connection)
+        public Task ClickAsync(HubConnection connection)
```

Comment:
Does a change link this need API review?

Reply:
This isn't a shipping package. We publish this to our nightly feed, so users could consume it, but it's not part of public API surface.